### DISCUSSION TITLE
Plan with Customization of pages

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1997] [Help] [answered]
Hi @mike12345567 what is the plan with the formating of the individual pages? It feels like we lost a lot of functionality. I can not get the new fields to look like the old fields, am I missing something in the new UI?  
for example, some of my fields have the text on the top (if it's new) and some have them on the side. I can not find the functionality to get them uniform.  another example is formating the buttons. I cant change the color on clicks as I did before. 

![image](https://user-images.githubusercontent.com/80058238/125866746-09ed99c6-e66a-4e2c-b4f0-82ec52eaab34.png)
![image](https://user-images.githubusercontent.com/80058238/125866785-aa6216ff-5832-4d30-acbd-c53aaf5fe1e0.png)

**Comment by mike12345567** [comment, marked as answer]
Hi @AndrewJRamay - this is a change that has been discussed quite heavily in discussion #1888.

We have plans for the future to augment the design process heavily with theming but for this to be possible we did need to make some changes to the design section.

**Reply by KernOil** [reply]
thank you for the quick reply, I must have missed this discussion, my apologies for the double post. I look forward to the new update that will help unify the UI look across the app.

**Reply by sergey-dme** [reply]
Hi @AndrewJRamay you can use Custom CSS it helps in most of the cases, and solves almost everything related to app look/UI. It was quite hard for me during first two days since design section was removed but now I have used to it. As for _hover_ and _active_ properties, unfortunately it's currently impossible. Very waiting for the implementation when it will be possible on theming level.

**Reply by shogunpurple** [reply]
You can track the status of the theming work here: https://github.com/Budibase/budibase/issues/1847

We look forward to releasing it and getting your feedback! You will also be able to build a much more dynamic UI with the work we are doing at the moment on dynamic properties. This will allow you to change colours and settings on button clicks and events - providing much more UI flexibility.
